app/src/main/java/com/swiezaki/swiezakimobile/network/error/ErrorResponse.kt

```diff
@@ -0,0 +1,11 @@
+package com.swiezaki.swiezakimobile.network.error
+
+const val UNKNOWN = -1
+
+//Todo: error response model will be probably changed after discussion about error format from backend
```

Comment:
I think that this comment is definitely not needed. If you want to say few words about the commit or fragment of code, you can always create a pull request and comment on your code in review or just make a description of pull request.

Reply:
Yeah, comment may be not necessary, but in my opinion in this particular case it's quite important. Error handling at this moment is not implemented and using this parts can provide us an unexpected behavior. 
Using Todos is not a bad practice, especially at the beginning of project, isn't it?